{
	"schema_version": "1.6.0",
	"id": "RLSA-2024:5530",
	"modified": "2024-08-21T14:54:44.638430Z",
	"published": "2024-08-21T14:52:19.401187Z",
	"related": [
		"CVE-2024-6345"
	],
	"summary": "Important: python-setuptools security update",
	"details": "The python-setuptools package provides a collection of enhancements to Python distribution utilities allowing convenient building and distribution of Python packages.\n\nSecurity Fix(es):\n\n* pypa/setuptools: Remote code execution via download functions in the package_index module in pypa/setuptools (CVE-2024-6345)\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.",
	"affected": [
		{
			"package": {
				"ecosystem": "Rocky Linux:8",
				"name": "python-setuptools",
				"purl": "pkg:rpm/rocky-linux/python-setuptools?distro=rocky-linux-8-x86-64&epoch=0"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:39.2.0-8.el8_10"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://storage.googleapis.com/resf-osv-data/RLSA-2024:5530.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://errata.rockylinux.org/RLSA-2024:5530"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2297771"
		}
	],
	"credits": [
		{
			"name": "Rocky Enterprise Software Foundation"
		},
		{
			"name": "Red Hat"
		}
	]
}
